src/util/image.ts

```diff
@@ -0,0 +1,48 @@
+import { UploadedFile } from 'express-fileupload';
+import { Response, Request, NextFunction } from 'express';
+import storage from 'azure-storage';
+
+const containerName = 'image-container';
+
+/**
+ * Upload multiple images into Azure 
+ * @param files images 
+ */
+export function uploadFiles(files: UploadedFile[], res: Response): string[] {
```

Comment:
I think this could use the function `uploadFile` a lot more, so there's less code duplication there. For example, if `uploadFile` returns a string, then `uploadFile` could just be `files.map(file => uploadFile(file))`. If you're concerned about running `storage.createBlobService` more than necessary, we could make `uploadFile` an inner function of `uploadFiles` that depends on `uploadFiles` already creating the `blobSVC`. I was also thinking that it could take `UploadedFile | UploadedFile[]` so the code above doesn't have to do the check of whether it's an `UploadedFile` or `UploadedFile[]`.

Also, instead of taking `res` here and returning a response with a 500 status code, we can throw an error if it fails.

If you're not sure what I mean, I can probably refactor it pretty quick tomorrow or the day after.

Reply:
So the `UploadedFile | UploadedFile[]` is a little weird or maybe I'm not too familiar with dealing with typescript types, because arrays have attributes we can access like length for example so my for loop doesn't work if the passed in parameter is an UploadedFile as it doesn't have length so there has to be type checking in this case. Yeah, I'm not too sure what you exactly mean with the inner function, but I understand the code duplication concerns.